notesWindow.py

```diff
@@ -0,0 +1,118 @@
+from PyQt5.QtWidgets import QApplication,QWidget,QLabel,QTextEdit,QVBoxLayout,QGridLayout,QPushButton,QLineEdit,QSizePolicy,QScrollArea,QMessageBox
+from PyQt5.QtGui import QIcon,QImage,QPixmap
+from PyQt5.QtCore import Qt
+import sys
+
+from asset.modules.notesLib.authenticator import createFile,getGrootFolderContent
+
+class noteCreationBox(QWidget):
+ 
+    def __init__(self):
+        super(noteCreationBox,self).__init__()
+        self.title = 'Note Maker Google Docs'
+        self.initUI()
+        
+    def initUI(self):
+        self.setWindowTitle(self.title)
+        self.Layout = noteCreationWidgetLayout()
+        self.setLayout(self.Layout.noteLayout)
+        self.setGeometry(150,150,800,400)
+        self.show()
+
+
+class noteCreationWidgetLayout(QWidget):
+    def __init__(self,parent=None):
+        super(noteCreationWidgetLayout,self).__init__()
+        
+        #Elements Present on Note Creation GUI
+        self.title = QLabel('Title')
+        self.titleEdit = QLineEdit()
+        self.summary = QLabel('Summary')
+        self.summaryEdit = QTextEdit()
+        self.saveButton = QPushButton("Save")
+        self.saveButton.clicked.connect(self.createNewFile)
+        self.notesList = QPushButton("Notes")
+        self.notesList.clicked.connect(self.createNotesList)
+
+        #Main Layout
+        self.noteLayout = QGridLayout()
+
+        #Position of Widgets/Elements on Main Layout
+        self.noteLayout.addWidget(self.title, 1, 0)
+        self.noteLayout.addWidget(self.titleEdit, 1, 1)
+
+        self.noteLayout.addWidget(self.summary, 2, 0)
+        self.noteLayout.addWidget(self.summaryEdit, 2, 1, 10, 1)
+        
+        self.noteLayout.addWidget(self.saveButton,12,0)
+        self.noteLayout.addWidget(self.notesList,13,0)
+
+    def createNewFile(self):
+        """File is created in Groot/ in Google Drive
+        heading: file with title in the GUI is passed as heading
+        summart: file content"""
```

Comment:
I think it will be `summary` instead of `summart`.

Reply:
Done